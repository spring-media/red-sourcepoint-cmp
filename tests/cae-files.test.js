const { existsSync, readdirSync, readFileSync } = require('fs');
const { parse, resolve } = require('path');

const prepareTests = () => {
  const caeDir = '.cae';

  if (!existsSync(caeDir)) {
    return [];
  }

  return readdirSync(caeDir)
    .map((file) => parse(file))
    .filter(({ ext }) => ['.css', '.html', '.json'].includes(ext))
    .map(({ base }) => ({ base, path: resolve(caeDir, base) }))
    .map((entry) => ({ name: entry.base, content: readFileSync(entry.path, 'utf8') }))
    .map((entry) => [entry.name, entry.content]);
};

describe('Testsuite CAE files', () => {
  it.each([...prepareTests()])('%s should have the expected content', (name, content) => {
    expect(content).toMatchSnapshot();
  });
});
