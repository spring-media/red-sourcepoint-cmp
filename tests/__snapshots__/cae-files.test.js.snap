// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testsuite CAE files red-cmp-components.css should have the expected content 1`] = `".embed-placeholder__container{background:#e9ecef;border-radius:8px;padding:12px;max-width:500px;margin:0 auto}.embed-placeholder__header{background:#fff;border:1px solid #ced4da;box-sizing:border-box;border-radius:8px;padding:12px;margin-bottom:10px}.embed-placeholder__headline{font-family:Gotham;font-style:normal;font-weight:900;font-size:16px;line-height:19px;color:#000;margin-bottom:6px}.embed-placeholder__description{font-family:Gotham XNarrow;font-style:normal;font-weight:400;font-size:16px;line-height:130%;color:#212529;margin-bottom:10px}.embed-placeholder__button{display:block;padding:9px 50px;width:100%;background:#00c373;border-radius:8px;border-style:none;margin-bottom:10px;font-family:Gotham XNarrow;font-style:normal;font-weight:700;font-size:15px;line-height:18px;letter-spacing:.03em;text-transform:uppercase;color:#fff}.embed-placeholder__footer-text{font-family:Gotham XNarrow;font-style:normal;font-weight:400;font-size:12px;line-height:130%;color:#212529}.embed-placeholder__text-link{text-decoration:underline;color:#212529}.social-sharing-popup__container{background:#343a40;border-radius:8px 8px 0 0;padding:16px;max-width:360px;width:100%}.social-sharing-popup__headline{font-family:Gotham,'Avenir Next','Helvetica Neue',sans-serif;font-style:normal;font-weight:800;font-size:16px;line-height:19px;color:#f8f9fa}.social-sharing-popup__description{font-family:'Gotham XNarrow','Avenir Next Condensed',sans-serif;font-weight:400;font-style:normal;font-size:16px;line-height:21px;color:#ced4da;margin-top:4px}.social-sharing-popup__description a{color:#ced4da;text-decoration:none;border-bottom:1px solid #ced4da}.social-sharing-popup__button-container{display:flex;justify-content:space-between;margin-top:16px}.social-sharing-popup__button{width:156px;height:36px;border-radius:8px;text-align:center;border:none;font-family:'Gotham XNarrow','Avenir Next Condensed',sans-serif;font-style:normal;font-weight:700;font-size:15px;line-height:18px;letter-spacing:.03em;text-transform:uppercase;color:#f8f9fa;cursor:pointer}.social-sharing-popup__button--close{background-color:transparent;border:2px solid #495057}.social-sharing-popup__button--accept{background-color:#00c373}"`;

exports[`Testsuite CAE files red-cmp-embed-facebook-placeholder.html should have the expected content 1`] = `
"<div data-server-rendered=\\"true\\" class=\\"embed-placeholder__container\\"><div class=\\"embed-placeholder__header\\"><svg width=\\"100%\\" height=\\"84\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><circle cx=\\"16\\" cy=\\"16\\" r=\\"16\\" fill=\\"#E9ECEF\\"></circle> <rect x=\\"40\\" y=\\"7\\" width=\\"60\\" height=\\"10\\" fill=\\"#E9ECEF\\"></rect> <rect x=\\"40\\" y=\\"19\\" width=\\"40\\" height=\\"6\\" fill=\\"#E9ECEF\\"></rect> <rect y=\\"44\\" width=\\"100%\\" height=\\"8\\" fill=\\"#E9ECEF\\"></rect> <rect y=\\"60\\" width=\\"100%\\" height=\\"8\\" fill=\\"#E9ECEF\\"></rect> <rect y=\\"76\\" width=\\"75%\\" height=\\"8\\" fill=\\"#E9ECEF\\"></rect> <path d=\\"M20 10.0611C20 4.50451 15.5229 0 10 0C4.47715 0 0 4.50451 0 10.0611C0 15.0828 3.65684 19.2452 8.4375 20V12.9694H5.89844V10.0611H8.4375V7.84452C8.4375 5.32296 9.93047 3.93012 12.2146 3.93012C13.3088 3.93012 14.4531 4.12663 14.4531 4.12663V6.60261H13.1922C11.95 6.60261 11.5625 7.37822 11.5625 8.17387V10.0611H14.3359L13.8926 12.9694H11.5625V20C16.3432 19.2452 20 15.0828 20 10.0611Z\\" fill=\\"#1877F2\\" style=\\"transform:translateX(calc(100% - 20px));\\"></path></svg></div> <div class=\\"embed-placeholder__headline\\">
    An dieser Stelle findest du Inhalte aus Facebook
  </div> <div class=\\"embed-placeholder__description\\">
    Um mit Inhalten aus Facebook und anderen sozialen Netzwerken zu interagieren oder diese darzustellen, brauchen wir
    deine Zustimmung.
  </div> <button class=\\"embed-placeholder__button\\">
    soziale Netzwerke aktivieren
  </button> <div class=\\"embed-placeholder__footer-text\\">
        Ich bin damit einverstanden, dass mir Inhalte von Drittanbietern angezeigt werden. Damit können
        personenbezogene Daten an Drittanbieter übermittelt werden. Dazu ist ggf. die Speicherung von Cookies auf
        deinem Gerät notwendig. Mehr Informationen dazu findest du
        <a href=\\"#\\" rel=\\"noopener\\" target=\\"_blank\\" class=\\"embed-placeholder__text-link\\">hier</a>.
      </div></div>"
`;

exports[`Testsuite CAE files red-cmp-embed-instagram-placeholder.html should have the expected content 1`] = `
"<div data-server-rendered=\\"true\\" class=\\"embed-placeholder__container\\"><div class=\\"embed-placeholder__header\\"><svg width=\\"100%\\" height=\\"312\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><circle cx=\\"12\\" cy=\\"12\\" r=\\"12\\" fill=\\"#E9ECEF\\"></circle> <rect x=\\"32\\" y=\\"7\\" width=\\"60\\" height=\\"10\\" fill=\\"#E9ECEF\\"></rect> <rect y=\\"32\\" width=\\"100%\\" height=\\"100%\\" fill=\\"#E9ECEF\\" style=\\"height:calc(100% - 32px);\\"></rect> <path d=\\"M11.6001 0H28.4001C34.8001 0 40.0001 5.2 40.0001 11.6V28.4C40.0001 31.4765 38.778 34.427 36.6026 36.6024C34.4271 38.7779 31.4766 40 28.4001 40H11.6001C5.20012 40 0.00012207 34.8 0.00012207 28.4V11.6C0.00012207 8.52349 1.22226 5.57298 3.39768 3.39756C5.57311 1.22214 8.52361 0 11.6001 0ZM11.2001 4C9.29056 4 7.45922 4.75857 6.10895 6.10883C4.75869 7.45909 4.00012 9.29044 4.00012 11.2V28.8C4.00012 32.78 7.22012 36 11.2001 36H28.8001C30.7097 36 32.541 35.2414 33.8913 33.8912C35.2416 32.5409 36.0001 30.7096 36.0001 28.8V11.2C36.0001 7.22 32.7801 4 28.8001 4H11.2001ZM30.5001 7C31.1632 7 31.799 7.26339 32.2679 7.73223C32.7367 8.20107 33.0001 8.83696 33.0001 9.5C33.0001 10.163 32.7367 10.7989 32.2679 11.2678C31.799 11.7366 31.1632 12 30.5001 12C29.8371 12 29.2012 11.7366 28.7324 11.2678C28.2635 10.7989 28.0001 10.163 28.0001 9.5C28.0001 8.83696 28.2635 8.20107 28.7324 7.73223C29.2012 7.26339 29.8371 7 30.5001 7ZM20.0001 10C22.6523 10 25.1958 11.0536 27.0712 12.9289C28.9466 14.8043 30.0001 17.3478 30.0001 20C30.0001 22.6522 28.9466 25.1957 27.0712 27.0711C25.1958 28.9464 22.6523 30 20.0001 30C17.348 30 14.8044 28.9464 12.9291 27.0711C11.0537 25.1957 10.0001 22.6522 10.0001 20C10.0001 17.3478 11.0537 14.8043 12.9291 12.9289C14.8044 11.0536 17.348 10 20.0001 10ZM20.0001 14C18.4088 14 16.8827 14.6321 15.7575 15.7574C14.6323 16.8826 14.0001 18.4087 14.0001 20C14.0001 21.5913 14.6323 23.1174 15.7575 24.2426C16.8827 25.3679 18.4088 26 20.0001 26C21.5914 26 23.1175 25.3679 24.2428 24.2426C25.368 23.1174 26.0001 21.5913 26.0001 20C26.0001 18.4087 25.368 16.8826 24.2428 15.7574C23.1175 14.6321 21.5914 14 20.0001 14Z\\" fill=\\"#868E96\\" style=\\"transform:translate(calc(50% - 20px), 50%);\\"></path> <path d=\\"M5.80006 0H14.2001C17.4001 0 20.0001 2.6 20.0001 5.8V14.2C20.0001 15.7383 19.389 17.2135 18.3013 18.3012C17.2136 19.3889 15.7383 20 14.2001 20H5.80006C2.60006 20 6.10352e-05 17.4 6.10352e-05 14.2V5.8C6.10352e-05 4.26174 0.611131 2.78649 1.69884 1.69878C2.78655 0.61107 4.26181 0 5.80006 0ZM5.60006 2C4.64528 2 3.72961 2.37928 3.05448 3.05442C2.37935 3.72955 2.00006 4.64522 2.00006 5.6V14.4C2.00006 16.39 3.61006 18 5.60006 18H14.4001C15.3548 18 16.2705 17.6207 16.9456 16.9456C17.6208 16.2705 18.0001 15.3548 18.0001 14.4V5.6C18.0001 3.61 16.3901 2 14.4001 2H5.60006ZM15.2501 3.5C15.5816 3.5 15.8995 3.6317 16.1339 3.86612C16.3684 4.10054 16.5001 4.41848 16.5001 4.75C16.5001 5.08152 16.3684 5.39946 16.1339 5.63388C15.8995 5.8683 15.5816 6 15.2501 6C14.9185 6 14.6006 5.8683 14.3662 5.63388C14.1318 5.39946 14.0001 5.08152 14.0001 4.75C14.0001 4.41848 14.1318 4.10054 14.3662 3.86612C14.6006 3.6317 14.9185 3.5 15.2501 3.5ZM10.0001 5C11.3261 5 12.5979 5.52678 13.5356 6.46447C14.4733 7.40215 15.0001 8.67392 15.0001 10C15.0001 11.3261 14.4733 12.5979 13.5356 13.5355C12.5979 14.4732 11.3261 15 10.0001 15C8.67398 15 7.40221 14.4732 6.46453 13.5355C5.52684 12.5979 5.00006 11.3261 5.00006 10C5.00006 8.67392 5.52684 7.40215 6.46453 6.46447C7.40221 5.52678 8.67398 5 10.0001 5ZM10.0001 7C9.20441 7 8.44135 7.31607 7.87874 7.87868C7.31613 8.44129 7.00006 9.20435 7.00006 10C7.00006 10.7956 7.31613 11.5587 7.87874 12.1213C8.44135 12.6839 9.20441 13 10.0001 13C10.7957 13 11.5588 12.6839 12.1214 12.1213C12.684 11.5587 13.0001 10.7956 13.0001 10C13.0001 9.20435 12.684 8.44129 12.1214 7.87868C11.5588 7.31607 10.7957 7 10.0001 7Z\\" fill=\\"black\\" style=\\"transform:translateX(calc(100% - 20px));\\"></path></svg></div> <div class=\\"embed-placeholder__headline\\">
    An dieser Stelle findest du Inhalte aus Instagram
  </div> <div class=\\"embed-placeholder__description\\">
    Um mit Inhalten aus Instagram und anderen sozialen Netzwerken zu interagieren oder diese darzustellen, brauchen
    wir deine Zustimmung.
  </div> <button class=\\"embed-placeholder__button\\">
    soziale Netzwerke aktivieren
  </button> <div class=\\"embed-placeholder__footer-text\\">
        Ich bin damit einverstanden, dass mir Inhalte von Drittanbietern angezeigt werden. Damit können
        personenbezogene Daten an Drittanbieter übermittelt werden. Dazu ist ggf. die Speicherung von Cookies auf
        deinem Gerät notwendig. Mehr Informationen dazu findest du
        <a href=\\"#\\" rel=\\"noopener\\" target=\\"_blank\\" class=\\"embed-placeholder__text-link\\">hier</a>.
      </div></div>"
`;

exports[`Testsuite CAE files red-cmp-embed-placeholder.html should have the expected content 1`] = `
"<div data-server-rendered=\\"true\\" class=\\"embed-placeholder__container\\"><div class=\\"embed-placeholder__header\\"><svg width=\\"100%\\" height=\\"265\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><rect x=\\"38\\" y=\\"62\\" width=\\"81\\" height=\\"8\\" fill=\\"#E9ECEF\\"></rect> <rect x=\\"38\\" y=\\"88\\" width=\\"81\\" height=\\"8\\" fill=\\"#E9ECEF\\"></rect> <rect x=\\"38\\" y=\\"75\\" width=\\"61\\" height=\\"8\\" fill=\\"#E9ECEF\\"></rect> <rect x=\\"38\\" y=\\"173\\" width=\\"81\\" height=\\"8\\" fill=\\"#E9ECEF\\"></rect> <rect x=\\"38\\" y=\\"199\\" width=\\"81\\" height=\\"8\\" fill=\\"#E9ECEF\\"></rect> <rect x=\\"38\\" y=\\"186\\" width=\\"61\\" height=\\"8\\" fill=\\"#E9ECEF\\"></rect> <path d=\\"M289.782 110.125L313 78L343 78\\" stroke=\\"#868E96\\" stroke-width=\\"3\\"></path> <path d=\\"M287.287 170.75C292.847 161.12 296.117 150.339 296.845 139.242C297.572 128.145 295.737 117.03 291.481 106.756C287.226 96.4817 280.663 87.3243 272.303 79.9919C263.942 72.6596 254.006 67.3489 243.264 64.4707L239.511 78.4766C248.105 80.7792 256.053 85.0278 262.742 90.8936C269.431 96.7595 274.68 104.085 278.085 112.305C281.49 120.524 282.958 129.416 282.376 138.294C281.794 147.171 279.178 155.796 274.729 163.5L287.287 170.75Z\\" fill=\\"#868E96\\"></path> <path d=\\"M241.671 64.0628C230.542 61.3497 218.928 61.3124 207.781 63.954C196.635 66.5956 186.273 71.8409 177.545 79.26C168.817 86.6791 161.971 96.0608 157.568 106.636C153.166 117.212 151.332 128.68 152.217 140.101L166.673 138.981C165.965 129.844 167.433 120.669 170.955 112.209C174.477 103.749 179.953 96.2433 186.936 90.308C193.918 84.3727 202.208 80.1765 211.125 78.0632C220.042 75.9499 229.333 75.9797 238.237 78.1502L241.671 64.0628Z\\" fill=\\"#CED4DA\\"></path> <path d=\\"M152.396 142.061C153.976 157.128 160.236 171.321 170.296 182.647C180.357 193.974 193.713 201.864 208.489 205.209C223.264 208.555 238.716 207.188 252.675 201.301C266.633 195.414 278.397 185.302 286.314 172.386L273.951 164.808C267.618 175.142 258.207 183.231 247.04 187.941C235.873 192.651 223.511 193.744 211.691 191.067C199.871 188.391 189.186 182.079 181.137 173.018C173.088 163.957 168.08 152.602 166.816 140.549L152.396 142.061Z\\" fill=\\"#CED4DA\\"></path> <rect x=\\"349\\" y=\\"62\\" width=\\"81\\" height=\\"8\\" fill=\\"#868E96\\"></rect> <rect x=\\"349\\" y=\\"88\\" width=\\"81\\" height=\\"8\\" fill=\\"#868E96\\"></rect> <rect x=\\"349\\" y=\\"75\\" width=\\"61\\" height=\\"8\\" fill=\\"#868E96\\"></rect></svg></div> <div class=\\"embed-placeholder__headline\\">
      An dieser Stelle findest du Inhalte von Drittanbietern
    </div> <div class=\\"embed-placeholder__description\\">
      Um mit Inhalten von Drittanbietern zu interagieren oder diese darzustellen, brauchen wir deine Zustimmung.
    </div> <button class=\\"embed-placeholder__button\\">
          externen Inhalt aktivieren
        </button> <div class=\\"embed-placeholder__footer-text\\">
        Ich bin damit einverstanden, dass mir Inhalte von Drittanbietern angezeigt werden. Damit können
        personenbezogene Daten an Drittanbieter übermittelt werden. Dazu ist ggf. die Speicherung von Cookies auf
        deinem Gerät notwendig. Mehr Informationen dazu findest du
        <a href=\\"#\\" rel=\\"noopener\\" target=\\"_blank\\" class=\\"embed-placeholder__text-link\\">hier</a>.
      </div></div>"
`;

exports[`Testsuite CAE files red-cmp-embed-social-networks-placeholder.html should have the expected content 1`] = `
"<div data-server-rendered=\\"true\\" class=\\"embed-placeholder__container\\"><div class=\\"embed-placeholder__header\\"><svg width=\\"100%\\" height=\\"84\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><circle cx=\\"16\\" cy=\\"16\\" r=\\"16\\" fill=\\"#E9ECEF\\"></circle> <rect x=\\"40\\" y=\\"7\\" width=\\"60\\" height=\\"10\\" fill=\\"#E9ECEF\\"></rect> <rect x=\\"40\\" y=\\"19\\" width=\\"40\\" height=\\"6\\" fill=\\"#E9ECEF\\"></rect> <rect y=\\"44\\" width=\\"100%\\" height=\\"8\\" fill=\\"#E9ECEF\\"></rect> <rect y=\\"60\\" width=\\"100%\\" height=\\"8\\" fill=\\"#E9ECEF\\"></rect> <rect y=\\"76\\" width=\\"75%\\" height=\\"8\\" fill=\\"#E9ECEF\\"></rect></svg></div> <div class=\\"embed-placeholder__headline\\">
      An dieser Stelle findest du Inhalte aus sozialen Netzwerken
    </div> <div class=\\"embed-placeholder__description\\">
      Um mit Inhalten aus sozialen Netzwerken zu interagieren oder diese darzustellen, brauchen wir deine Zustimmung.
    </div> <button class=\\"embed-placeholder__button\\">
    soziale Netzwerke aktivieren
  </button> <div class=\\"embed-placeholder__footer-text\\">
        Ich bin damit einverstanden, dass mir Inhalte von Drittanbietern angezeigt werden. Damit können
        personenbezogene Daten an Drittanbieter übermittelt werden. Dazu ist ggf. die Speicherung von Cookies auf
        deinem Gerät notwendig. Mehr Informationen dazu findest du
        <a href=\\"#\\" rel=\\"noopener\\" target=\\"_blank\\" class=\\"embed-placeholder__text-link\\">hier</a>.
      </div></div>"
`;

exports[`Testsuite CAE files red-cmp-embed-twitter-placeholder.html should have the expected content 1`] = `
"<div data-server-rendered=\\"true\\" class=\\"embed-placeholder__container\\"><div class=\\"embed-placeholder__header\\"><svg width=\\"100%\\" height=\\"84\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><circle cx=\\"16\\" cy=\\"16\\" r=\\"16\\" fill=\\"#E9ECEF\\"></circle> <rect x=\\"40\\" y=\\"7\\" width=\\"60\\" height=\\"10\\" fill=\\"#E9ECEF\\"></rect> <rect x=\\"40\\" y=\\"19\\" width=\\"40\\" height=\\"6\\" fill=\\"#E9ECEF\\"></rect> <rect y=\\"44\\" width=\\"100%\\" height=\\"8\\" fill=\\"#E9ECEF\\"></rect> <rect y=\\"60\\" width=\\"100%\\" height=\\"8\\" fill=\\"#E9ECEF\\"></rect> <rect y=\\"76\\" width=\\"75%\\" height=\\"8\\" fill=\\"#E9ECEF\\"></rect> <path d=\\"M19.6 1.8456C18.8952 2.1616 18.1344 2.3704 17.3392 2.464C18.1536 1.9768 18.7752 1.204 19.072 0.284C18.3112 0.7336 17.4664 1.0656 16.5704 1.24C15.8504 0.476 14.8272 0 13.6952 0C11.52 0 9.75762 1.7624 9.75762 3.936C9.75762 4.2448 9.79202 4.5448 9.85922 4.8328C6.58562 4.6672 3.68402 3.1008 1.74242 0.716C1.40482 1.2976 1.20962 1.9752 1.20962 2.696C1.20962 4.0616 1.90322 5.268 2.96082 5.9736C2.31522 5.956 1.70882 5.7744 1.17682 5.4824C1.17682 5.4984 1.17682 5.5168 1.17682 5.532C1.17682 7.4408 2.53362 9.0336 4.33682 9.3936C4.00642 9.4832 3.65762 9.5336 3.29842 9.5336C3.04322 9.5336 2.79922 9.5064 2.55602 9.4616C3.05762 11.024 4.51202 12.1648 6.23442 12.1976C4.88402 13.2552 3.18882 13.8848 1.34082 13.8848C1.02242 13.8848 0.711224 13.8664 0.400024 13.8288C2.14722 14.9504 4.21682 15.5992 6.44082 15.5992C13.6848 15.5992 17.6472 9.596 17.6472 4.3928C17.6472 4.2184 17.644 4.0496 17.6352 3.8808C18.404 3.3304 19.072 2.6368 19.6 1.8456Z\\" fill=\\"#4AA1EC\\" style=\\"transform:translateX(calc(100% - 20px));\\"></path></svg></div> <div class=\\"embed-placeholder__headline\\">
    An dieser Stelle findest du Inhalte aus Twitter
  </div> <div class=\\"embed-placeholder__description\\">
    Um mit Inhalten aus Twitter und anderen sozialen Netzwerken zu interagieren oder diese darzustellen, brauchen wir
    deine Zustimmung.
  </div> <button class=\\"embed-placeholder__button\\">
    soziale Netzwerke aktivieren
  </button> <div class=\\"embed-placeholder__footer-text\\">
        Ich bin damit einverstanden, dass mir Inhalte von Drittanbietern angezeigt werden. Damit können
        personenbezogene Daten an Drittanbieter übermittelt werden. Dazu ist ggf. die Speicherung von Cookies auf
        deinem Gerät notwendig. Mehr Informationen dazu findest du
        <a href=\\"#\\" rel=\\"noopener\\" target=\\"_blank\\" class=\\"embed-placeholder__text-link\\">hier</a>.
      </div></div>"
`;

exports[`Testsuite CAE files red-cmp-embed-youtube-placeholder.html should have the expected content 1`] = `
"<div data-server-rendered=\\"true\\" class=\\"embed-placeholder__container\\"><div class=\\"embed-placeholder__header\\"><svg width=\\"100%\\" height=\\"172\\" fill=\\"none\\" xmlns=\\"http://www.w3.org/2000/svg\\"><g style=\\"transform:translate(calc(50% - 20px), calc(50% - 20px)) scale(0.42);\\"><path d=\\"M 118.9,13.3 C 117.5,8.1 113.4,4 108.2,2.6 98.7,0 60.7,0 60.7,0 60.7,0 22.7,0 13.2,2.5 8.1,3.9 3.9,8.1 2.5,13.3 0,22.8 0,42.5 0,42.5 0,42.5 0,62.3 2.5,71.7 3.9,76.9 8,81 13.2,82.4 22.8,85 60.7,85 60.7,85 c 0,0 38,0 47.5,-2.5 5.2,-1.4 9.3,-5.5 10.7,-10.7 2.5,-9.5 2.5,-29.2 2.5,-29.2 0,0 0.1,-19.8 -2.5,-29.3 z\\" fill=\\"#E9ECEF\\"></path> <polygon points=\\"80.2,42.5 48.6,24.3 48.6,60.7\\" fill=\\"#ffffff\\"></polygon></g></svg></div> <div class=\\"embed-placeholder__headline\\">
    An dieser Stelle findest du Inhalte aus YouTube
  </div> <div class=\\"embed-placeholder__description\\">
    Um mit Inhalten aus YouTube und anderen sozialen Netzwerken zu interagieren oder diese darzustellen, brauchen wir
    deine Zustimmung.
  </div> <button class=\\"embed-placeholder__button\\">
    soziale Netzwerke aktivieren
  </button> <div class=\\"embed-placeholder__footer-text\\">
        Ich bin damit einverstanden, dass mir Inhalte von Drittanbietern angezeigt werden. Damit können
        personenbezogene Daten an Drittanbieter übermittelt werden. Dazu ist ggf. die Speicherung von Cookies auf
        deinem Gerät notwendig. Mehr Informationen dazu findest du
        <a href=\\"#\\" rel=\\"noopener\\" target=\\"_blank\\" class=\\"embed-placeholder__text-link\\">hier</a>.
      </div></div>"
`;

exports[`Testsuite CAE files red-cmp-social-sharing-popup.html should have the expected content 1`] = `
"<div data-server-rendered=\\"true\\" class=\\"social-sharing-popup__container\\"><div class=\\"social-sharing-popup__headline\\">
        Deine Datensicherheit bei der Nutzung der Teilen-Funktion
      </div> <div class=\\"social-sharing-popup__description\\">
        Um diesen Artikel oder andere Inhalte über soziale Netzwerke zu teilen, brauchen wir deine Zustimmung für
        <a href=\\"#\\" rel=\\"noopener\\" target=\\"_blank\\">
          diesen Zweck der Datenverarbeitung
        </a></div> <div class=\\"social-sharing-popup__button-container\\"><button class=\\"social-sharing-popup__button social-sharing-popup__button--close\\">
          Schließen
        </button> <button class=\\"social-sharing-popup__button social-sharing-popup__button--accept\\">
            Akzeptieren
          </button></div></div>"
`;
